Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -171,6 +174,23 @@ public partial class HumanoidProfileEditor : Control
 
             #endregion Gender
 
+            #region Species
+
+            var options = prototypeManager.EnumeratePrototypes<SpeciesPrototype>().ToArray();
+            for (var i = 0; i < options.Length; i++)
+            {
+                CSpeciesButton.AddItem(options[i].Name, i);
```

Comment:
If you do AddItem, with the Id from the option, then you don't have to keep the options array around in the OnItemSelected

Reply:
I don't get what you mean, the only valid type for an Id is an int.

Reply:
I thought Id was an int, not a string my bad.

Reply:
Yea I see no way to do this without the array; id is an `int`.